alibi/explainers/tests/test_ale.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from sklearn.datasets import load_boston, load_iris
+from sklearn.linear_model import LinearRegression, LogisticRegression
+import pytest
+from alibi.explainers.ale import ale_num, adaptive_grid, get_quantiles
+from alibi.api.defaults import DEFAULT_DATA_ALE, DEFAULT_META_ALE
+
+
+@pytest.mark.parametrize("min_bin_points", [1, 4, 10])
+def test_ale_num_linear_regression(min_bin_points):
+    """
+    The slope of the ALE of linear regression should equal the learnt coefficients
+    """
+    X, y = load_boston(return_X_y=True)
+    lr = LinearRegression().fit(X, y)
+    for feature in range(X.shape[1]):
+        q, ale, _ = ale_num(lr.predict, X, feature=feature, min_bin_points=min_bin_points)
+        assert np.allclose((ale[-1] - ale[0]) / (X[:, feature].max() - X[:, feature].min()), lr.coef_[feature])
```

Comment:
NB: Check [this](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) out! Think this is also something we should standardize across our codebase.

Reply:
Agree!

Reply:
Let's start a testing standard doc to develop in time ...